examples/gfx/vulkan/2_vulkan_3d_cube/main.cpp

```diff
@@ -1,5 +1,5 @@
-#define OMNI_TYPES_MATRIX_COLLUM_MAJOR_ORDER
-#define OMNI_TYPES_MATRIX4X4_PROJECTION_MATRIX_INVERSE_Y_AXIS
+#define BEBONE_TYPES_MATRIX_COLLUM_MAJOR_ORDER

```

Comment:
Grammar error
Change `BEBONE_TYPES_MATRIX_COLLUM_MAJOR_ORDER` to `BEBONE_TYPES_MATRIX_COLUMN_MAJOR_ORDER`